doi-proxy/src/main/java/no/unit/nva/doi/CrossRefClient.java

```diff
@@ -31,23 +36,35 @@
     public static final int TIMEOUT_DURATION = 30;
     public static final String COULD_NOT_FIND_ENTRY_WITH_DOI = "Could not find entry with DOI:";
     public static final String UNKNOWN_ERROR_MESSAGE = "Something went wrong. StatusCode:";
-
-    private static final String DOI_EXAMPLES = "10.1000/182, https://doi.org/10.1000/182";
-    public static final String ILLEGAL_DOI_MESSAGE = "Illegal DOI:%s. Valid examples:" + DOI_EXAMPLES;
     public static final String FETCH_ERROR = "CrossRefClient failed while trying to fetch:";
     public static final String CROSSREF_USER_AGENT =
             "nva-fetch-doi/1.0 (https://github.com/BIBSYSDEV/nva-fetch-doi; mailto:support@unit.no)";
-
-    private final transient HttpClient httpClient;
+    public static final String ADDING_TOKEN_IN_HEADER =
+            "CrossRefApiPlusToken.isPresent() == true, adding token in header";
```

Comment:
CrossRer Api token is present

Reply:
Correct to somthing that looks like a sentence...